causalml/feature_selection/filters.py

```diff
@@ -337,11 +340,19 @@ def _filter_D_one_feature(self, data, feature_name, y_name,
 
         totalSize = len(data.index)
 
+        # impute null if enabled
+        if null_impute is not None:
+            data[feature_name] = SimpleImputer(missing_values=np.nan, strategy=null_impute).fit_transform(data[feature_name].values.reshape(-1, 1))
+        elif data[feature_name].isna().any():
+            raise Exception("Null values present al column '{}'. Please impute the null value or use null_impute paramter provided!!!".format(feature_name))
```

Comment:
nit: do you mean `present in column` and `parameter` ?

Reply:
@manojbalaji1 seems like it was a typo?

Reply:
My bad will it

Reply:
Fixed it!